action_plugins/textfsm.py

```diff
@@ -9,7 +9,7 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-import StringIO
+from io import StringIO
```

Comment:
StringIO should be imported from ansible.module_utils.six otherwise this will break Python2.x